ui/components/app/advanced-gas-controls/advanced-gas-controls-row.component.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Typography from '../../ui/typography/typography';
+import {
+  COLORS,
+  TEXT_ALIGN,
+  DISPLAY,
+  TYPOGRAPHY,
+  FONT_WEIGHT,
+} from '../../../helpers/constants/design-system';
+
+import NumericInput from '../../ui/numeric-input/numeric-input.component';
+import Tooltip from '../../ui/tooltip/tooltip';
+
+export default function AdvancedGasControlsRow({
+  titleText,
+  tooltipText,
+  titleDetailText,
+  error,
+}) {
+  return (
+    <div
+      className={classNames('advanced-gas-controls__row', {
+        'advanced-gas-controls__row--error': error,
+      })}
+    >
+      <label>
+        <div className="advanced-gas-controls__row-heading">
+          <div className="advanced-gas-controls__row-heading-title">
+            <Typography
+              tag={TYPOGRAPHY.H6}
+              display={DISPLAY.INLINE_BLOCK}
```

Comment:
Typography makes use of the 'Box' component. currently you can override the box element of the Typography component by passing boxProps={{ display: DISPLAY.INLINE_BLOCK }}

Reply:
Good shout!